src/components/Bookstore.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Calculator.css';
+
+const Bookstore = () => (
+  <Switch>
+    <Route exact path="/">
+      <div id="Books" />
+    </Route>
+    <Route path="/categories">
+      <h1>Under Construction</h1>
```

Comment:
- [ ] In App.js, the categories route should display the Categories component. So kindly create a Categories component and in it return the `Under construction` text.